docs/frontend.md

```diff
@@ -1 +1,51 @@
 # Frontend
+This site is built using Drupal and Federalist with a USWDS theme.
+
+## Theme commands
+These commands should be run inside web/themes/custom/vote_gov
+| **Command**      | **Use case**                                                                      |
+|------------------|-----------------------------------------------------------------------------------|
+| `npm run build`  | Build assets with gulp                                                            |
+| `npm run dev`    | Run scss linter                                                                   |
+
+## Theme file structure
+```
+web/
+├── core
+├── modules
+├── profiles
+├── sites
+└── themes/
+    └── custom/
+        └── vote_gov/
+            ├── ckeditor
+            ├── config
+            ├── dist
+            ├── example-settings
+            ├── fonts
+            ├── img
+            ├── php-includes
+            ├── js
+            ├── sass
+            └── templates
+```
+**ckeditor** - WYSIWYG overrides
+
+**config** - Theme configuration
+
+**dist** - All files generated by the gulp process and contains all image/font assets, minified css, and minified js. Changes made in these files will be overridden. 
+
+**example-settings** - Default theme example settings
+
+**fonts** - Custom fonts such as Navajo that compile into dist
+
+**img** - All favicons and logos that compile into dist
+
+**php-includes** - Preprocess functions organized by use type
+
+**js** - Javascript files that compile into dist
+
+**sass** - SCSS files that compile into dist
+
+**templates** - Vote.gov theme templates and template overrides
```

Comment:
We will need to clean out templates that are not in use. 

I think we should also call out specifically in this document that `partials` are templates that are being included or extended by other templates.